corehq/apps/callcenter/tasks.py

```diff
@@ -46,6 +46,17 @@ def calculate_indicators():
         indicator_set.get_data()
 
 
+def sync_webuser_usercases_if_applicable(user_id, spawn_task):
+    user = CouchUser.get_by_user_id(user_id)
+    for domain in user.get_domains():
+        domain_obj = Domain.get_by_name(domain)
+        if domain_obj.call_center_config.enabled or domain_obj.usercase_enabled:
+            if spawn_task:
+                sync_webuser_usercases_task.delay(user, domain_obj)
```

Comment:
Here you want to pass in the User ID and have the task re-fetch the user when it runs. This has 2 advantages:
1. the task get's a fresh copy of the user when it runs
2. we don't need to serialize the whole user object for the task